tests/interactive_test.py

```diff
@@ -0,0 +1,201 @@
+import logging
+
+logging.basicConfig(level=logging.DEBUG)
+
+import json
+from tempfile import gettempdir
+
+from spyql.query import Query
+from spyql import log
+from spyql.utils import join_paths
+
+
+def eq_test_nrows(query, expectation, **kwargs):
+    log.user_debug(f"----")
+    q = Query(query)
```

Comment:
🙃 Perhaps this variable's name in the multiple functions could be uniformized? Here it's called `q` (I know `query` was already taken by the argument name), there's a function where it's called `_q`. In most functions it's called `query`. Perhaps it can be called `q` everywhere?

Is there a reason behind the different naming? (not a blocker, it can stay as is if you'd like)

Reply:
No reason, just being lazy :-) 
Let's make it cleaner

Reply:
Actually, this was already done here: https://github.com/dcmoura/spyql/pull/68/commits/b66e451ab48189263ec85c0477cebe9740c68b33

Sorry, same issue as here: https://github.com/dcmoura/spyql/pull/68#discussion_r827524328